data-raw/process_data_sets_kara.R

```diff
@@ -0,0 +1,31 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#reading in csv
+nba_elo <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo.csv")
+nba_elo_latest <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo_latest.csv")
+
+rownames(nba_elo) <- NULL
+rownames(nba_elo_latest) <- NULL
+
+#tidying and taming data for nba_elo
+nba_elo %>% clean_names()
```

Comment:
you pipe `nba_elo` here into to `clean_names()`, but don't save it. As a resulting we have variable names that have the ugly tick marks. For example `carm-elo1_pre` should be `carm_elo1_pre`
```
> glimpse(nba_elo)
Observations: 69,636
Variables: 24
$ date             <date> 1946-11-01, 1946-11-02, 1946-11-02, 1946-11-02, 1946-11-…
$ season           <int> 1947, 1947, 1947, 1947, 1947, 1947, 1947, 1947, 1947, 194…
$ neutral          <int> 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, …
$ playoff          <chr> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ team1            <chr> "TRH", "CHS", "PRO", "STB", "DTF", "CLR", "PIT", "BOS", "…
$ team2            <chr> "NYK", "NYK", "BOS", "PIT", "WSC", "TRH", "WSC", "CHS", "…
$ elo1_pre         <dbl> 1300.000, 1300.000, 1300.000, 1300.000, 1300.000, 1300.00…
$ elo2_pre         <dbl> 1300.000, 1306.723, 1300.000, 1300.000, 1300.000, 1293.27…
$ elo_prob1        <dbl> 0.6400650, 0.6311012, 0.6400650, 0.6400650, 0.6400650, 0.…
$ elo_prob2        <dbl> 0.3599350, 0.3688988, 0.3599350, 0.3599350, 0.3599350, 0.…
$ elo1_post        <dbl> 1293.277, 1309.652, 1305.154, 1304.691, 1279.619, 1307.12…
$ elo2_post        <dbl> 1306.723, 1297.071, 1294.846, 1295.309, 1320.381, 1286.15…
$ `carm-elo1_pre`  <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ `carm-elo2_pre`  <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ `carm-elo_prob1` <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ `carm-elo_prob2` <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ `carm-elo1_post` <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ `carm-elo2_post` <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ raptor1_pre      <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ raptor2_pre      <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ raptor_prob1     <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ raptor_prob2     <dbl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, N…
$ score1           <int> 66, 63, 59, 56, 33, 71, 56, 55, 49, 81, 73, 63, 73, 76, 7…
$ score2           <int> 68, 47, 53, 51, 50, 60, 71, 57, 53, 75, 65, 68, 71, 66, 6…
```

Reply:
Could you also convert both `team` variables to factors?